lib/makara/cache.rb

```diff
@@ -16,10 +16,13 @@ def store=(store)
       end
 
       def read(key)
-        store.try(:read, key)
+        store.try(:read, key).tap do |value|
```

Comment:
I see. cool.
Nice to understand the approach. I'm super old school, but over all the gem is all ruby-fancy so I can get behind it :-)